include/oneapi/dpl/internal/reduce_by_segment_impl.h

```diff
@@ -215,13 +215,13 @@ __sycl_reduce_by_segment(_ExecutionPolicy&& __exec, _Range1&& __keys, _Range2&&
     using _CustomName = typename _Policy::kernel_name;
 
     using _SegReduceCountKernel = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_generator<
-        _SegReduceCountPhase, _CustomName, _Range1, _Range2, _Range3, _Range4, _BinaryPredicate, _BinaryOperator>;
+        _SegReduceCountPhase, _CustomName, _ExecutionPolicy, _Range1, _Range2, _Range3, _Range4, _BinaryPredicate, _BinaryOperator>;
```

Comment:
This is optional. Please, consider splitting this long line and other similar lines, which are longer than 120 symbols.

Reply:
I wanted to apply some clang-format suggestions anyway